js/views/ListingCard.js

```diff
@@ -66,7 +66,7 @@ export default class extends baseVw {
   }
 
   className() {
-    return 'listingCard col clrBr clrT clrP';
+    return 'listingCard col clrBr clrT clrP contentBox';
```

Comment:
We should remove the border style and border width on the listingCard class so we're not assigning the same properties twice.